Modules/private/CetOverrideFindPackage.cmake

```diff
@@ -64,8 +64,8 @@ X
 #   appropriate time.
 
 
-# Once only!
-include_guard(GLOBAL)
+# Once only per directory!
+include_guard(DIRECTORY)
```

Comment:
For consistency with the rest of Cetmodules, could you omit `DIRECTORY` (this is the default scope for `include_guard()`)?

Reply:
Done